src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ Notes](../../RELEASENOTES.md).
   now set the `TE=trailers` HTTP request header to improve interoperability.
   ([#6449](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6449))
 
+* Write `byte[]` to `bytes_value` instead of `array_value` in TagWriter. ([#6535](https://github.com/open-telemetry/opentelemetry-dotnet/issues/6535))
```

Comment:
```suggestion
* Improved performance exporting `byte[]` attributes as native binary format
  instead of arrays.
  ([#6535](https://github.com/open-telemetry/opentelemetry-dotnet/issues/6535))
```

Reply:
Use the PR link instead of the issue link to align with previous entries.